docs/installation/install-livepeer/binary-release.md

```diff
@@ -40,19 +39,37 @@ mv livepeer-linux-amd64/* /usr/local/bin/
 
 ### Windows
 
-```bash
-# <RELEASE_VERSION> is the release version i.e. 0.5.23
+.tar
+
+```shell
+# <RELEASE_VERSION> is the release version, e.g. v0.5.29
 
-# First, fetch the latest release
+# Fetch the latest release .tar.gz 
 wget https://github.com/livepeer/go-livepeer/releases/download/<RELEASE_VERSION>/livepeer-windows-amd64.tar.gz
 
 # Next, extract it
-tar -zxvf livepeer-windows-amd64.tar.gz
+tar -zxvf li2
```

Comment:
What is this `li2`? Should we keep `livepeer-windows-amd64.tar.gz`?

Reply:
no idea, that was on the version i had, maybe i changed it by accident. Thanks for pointing it out.

Reply:
yeah, I'd change it to `livepeer-windows-amd64.tar.gz`, `li2` seems like a mistake.